javarosa/src/main/java/org/javarosa/xpath/expr/XPathFuncExpr.java

```diff
@@ -170,15 +170,8 @@ public Object evalRaw(DataInstance model, EvaluationContext evalContext) {
         //TODO: Func handlers should be able to declare the desire for short circuiting as well
         if (name.equals("if") && args.length == 3) {
             return ifThenElse(model, evalContext, args, argVals);

```

Comment:
only comment is same as on the `cond` PR- are you sure there wasn't a reason for the `argVals` thing?